react/tic-tac-toe/src/app/components/Router/components/CheckRoute/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { Redirect, Route } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { string, func } from 'prop-types';
+
+class CheckRoute extends Component {
```

Comment:
I think this should be a function Component

Reply:
I have refactored that component and now is a `Function Component`